lsp-clients.el

```diff
@@ -659,5 +659,27 @@ finding the executable with `exec-path'."
                   :server-id 'fortls))
 
 
+
+;; Kotlin
+(defgroup lsp-kotlin nil
+  "Kotlin."
+  :group 'lsp-mode
+  :tag "Kotlin")
+
+(defcustom lsp-clients-kotlin-server-command
```

Comment:
this one is not used(you could remove the group as well).

Reply:
removed `lsp-clients-kotlin-server-commad`